exe/dirtree

```diff
@@ -24,13 +24,21 @@ OptionParser.new do |opts|
   ) do
     puts opts
     exit
+end
+
+  opts.on(
+    '-l',
+    '--local-dependencies', 
+    'Use saved JavaScript libraries instead of downloadign them'
+   ) do
+    options[:local_dependencies] = true
   end
 
   opts.on(
-    '-oFile.html',
-    '--output=File.html',
-    'Specify a path to write output, by default will be printed to STDOUT'
-  ) do |value|
+    '-oFile.html', 
+    '--output=File.html', 
+    'Specify a path to write output, if not specified output will be printed to STDOUT'
+   ) do |value|
```

Comment:
any reason we need spaces at the end of these lines? 
also make sure lines are not long "rubocop" should be green for these changes